devstack/settings

```diff
@@ -0,0 +1,66 @@
+#                                                                      -*-sh-*-
+# Devstack settings that are wanted when using Calico
+# ===================================================
+
+# Disable services that Devstack starts by default, but that we don't
+# need for Calico networking demonstration: 'dstat' is a rolling
+# display of memory and CPU usage by the various OpenStack services
+# and agents; 'tempest' is OpenStack's high level testing framework;
+# and 'n-net' is Nova networking.
+disable_service dstat
```

Comment:
Are we actually incompatible with dstat and tempest?  Seems odd to blacklist them just because they're not so relevant to the particular demo.